kubeflow/common/asm/options/ingressgateway-annotations.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This overlay adds compatibility for upgrades to clusters using GCLB
+# for certain versions of istioctl
+
+---
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
```

Comment:
I'd suggest renaming this file to `ingressgateway-iap.yaml` or sth similar

IAP is a more important keyword in this patch than annotations.

Reply:
This makes sense, I update the name to `ingressgateway-iap.yaml`.